var/spack/repos/builtin/packages/nvtop/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Nvtop(CMakePackage):
+    """Nvtop stands for NVidia TOP, a (h)top like task monitor for NVIDIA GPUs.
+    It can handle multiple GPUs and print information about them
+    in a htop familiar way."""
+
+    homepage = "https://github.com/Syllo/nvtop"
+    url      = "https://github.com/Syllo/nvtop/archive/1.1.0.tar.gz"
+
+    version('1.1.0', sha256='00470cde8fc48d5a5ed7c96402607e474414d94b562b21189bdde1dbe6b1d1f3')
+
+    depends_on('cmake')
+    depends_on('ncurses')
+    depends_on('git')
+    depends_on('cuda')
```

Comment:
Would it make sense to inherit also from `CudaPackage` and drop this dependency?

Reply:
I added CudaPackage as a class and tried installing again. There was an error that I think has something to do with the cmake nvml part, but not quite sure what that means yet. 

`CMakeFiles/nvtop.dir/extract_gpuinfo.c.o: In function `update_graphical_process':
/tmp/root/spack-stage/spack-stage-nvtop-1.1.0-u3qhvtr25atty7xujgix2aku3vhem6lz/spack-src/src/extract_gpuinfo.c:214: undefined reference to `nvmlDeviceGetGraphicsRunningProcesses_v2'
CMakeFiles/nvtop.dir/extract_gpuinfo.c.o: In function `update_compute_process':
/tmp/root/spack-stage/spack-stage-nvtop-1.1.0-u3qhvtr25atty7xujgix2aku3vhem6lz/spack-src/src/extract_gpuinfo.c:246: undefined reference to `nvmlDeviceGetComputeRunningProcesses_v2'`